packages/server/graphql/mutations/helpers/updateUserProfile.ts

```diff
@@ -38,7 +37,8 @@ const updateUserProfile = async (
 
   if (validUpdatedUser.picture && validUpdatedUser.picture.endsWith('.svg')) {
     const res = await fetch(validUpdatedUser.picture)
-    const buffer = await res.buffer()
+    const arrayBuffer = await res.arrayBuffer()
```

Comment:
this is the only real change

Reply:
right you are! we got some good spaghetti code going on there.
shall i remove that `.svg` conditional clause in this PR or should we save it for later? OK with either option.

Reply:
Let's leave it for later and then let's remove the `picture` input from this mutation completely.

Reply:
Issue made! https://github.com/ParabolInc/parabol/issues/8207